package.json

```diff
@@ -80,6 +80,8 @@
     "script:gen-templates": "pnpm runts ./scripts/generate-template-variations.ts",
     "script:list-published": "pnpm runts scripts/lib/getPackageRegistryVersions.ts",
     "script:pack": "pnpm runts scripts/pack-all-to-dest.ts",
+    "template:blank": "cd templates/blank && cross-env NODE_OPTIONS=--no-deprecation pnpm --ignore-workspace install && cp .env .env.example || true && pnpm dev",
```

Comment:
In order to be windows-compatible, we should be using `copyfiles` here.

Reply:
Also, I'm a bit confused why we're copying `.env` to `.env.example` and not the other way around.